src/js/components/Carousel/index.d.ts

```diff
@@ -7,6 +7,7 @@ export interface CarouselProps {
   margin?: "none" | "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | {bottom?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string,horizontal?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string,left?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string,right?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string,top?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string,vertical?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string} | string;
   fill?: boolean;
   play?: number;
+  controls?: "true" | "false" | "arrows" | "selectors"
```

Comment:
thanks for adding the typescript definition, we are trying to maintain alphabetical order, I know the rest of the file isn't perfect but it will be helpful if we can fix it step by step and moving this line to line #6.

Reply:
@ShimiSun yeah sure, sorry I didn`t notice the order, one moment going to correct it